app/models/card.rb

```diff
@@ -1,7 +1,18 @@
 class Card < ActiveRecord::Base
-  before_create :set_review
+  before_validation :set_review_date, on: :create
+  validate :original_translated_text_equal
+  validates :original_text, :translated_text, :review_date, presence: true
 
-  def set_review
-    self.review = Time.now + 3.days
-  end
+  protected
+    def set_review_date
+      self.review_date = Time.now + 3.days
+    end
+
+    def original_translated_text_equal
+      self.original_text = original_text.mb_chars.downcase.to_s.squeeze(' ').lstrip
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.